conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -155,23 +155,18 @@ func (algodImp *algodImporter) GetBlock(rnd uint64) (data.BlockData, error) {
 	var blk data.BlockData
 
 	for retries := 0; retries < 3; retries++ {
-		// nextRound - 1 because the endpoint waits until the subsequent block is committed to return
```

Comment:
I'm wondering if this retry loop is necessary. there's already retries at the pipeline level. in this loop there's no wait before the next try, could this loop always finish before the next block is available?

Reply:
Yep, just added retires w/ backoff to the most recent version.
```
for r := 0; r < retries; r++ {
		time.Sleep(time.Duration(waitMultiplier*float64(r)) * initialWait)
...
```

We could just rely on the pipeline-level retires like you said, but retries with backoff make more sense in the case of the follower node since the catchup service gets started/stopped so frequently.